File: Tests/test11/median_number.pl
#!/usr/bin/perl -w

use strict;

my @sortedList = sort { $a <=> $b } @ARGV;
my $sizeOfList = @sortedList;
my $count = 1;

foreach my $arg (@sortedList) {
	if ($count == ($sizeOfList/2)+.5) {
		print "$arg\n";
	}
	$count++;
}

File: Tests/test11/median_number.py
#!/usr/bin/python
import sys

argList = []
count = 1

for i in sys.argv[1:]:
	argList.append(i)

argList.sort(key=int)
listSize = len(argList)

for i in argList:
	if (count == (listSize/2)+1):
		print (i)
	count += 1

